backend/src/controllers/item.js

```diff
@@ -0,0 +1,114 @@
+const express = require('express');
+const _ = require('lodash');
+const buyerView = ['item_name', 'pic_url', 'item_description', 'tags', 'sale_price', 'ticket_price', 'total_tickets', 'status', 'deadline']
+
+const ItemController = (itemModel, authService) => {
+  const router = express.Router();
+
+  // The API path to create a new item. Creates an item object and returns that object
+  router.post('/create', async (req, res) => {
+    if (!req.body) return res.status(400).json({
+      "message": "Malformed Request",
+    });
+    body = req.body
+
+    const item_name = body['item_name']
+    const pic_url = body['pic_url']
+    const item_description = body['item_description']
+    const tags = body['tags']
+    const sale_price = body['sale_price']
+    const ticket_price = body['ticket_price']
+    const total_tickets = body['total_tickets']
+    const bids = body['bids']
+    const deadline = body['deadline']
+    const status = body['status']
+    const current_ledger = body['current_ledger']
```

Comment:
I feel like most of these things are things the backend should just set. Like sale price and number of tickets are set on the frontend but then ticket price is just found by division on backend. Bids starts empty and isnt passed by frontend. Deadline is set to be a week from now, status is set to In Progress or whatever we decide it should be when the item is created, and current ledger is 0. We only want the frontend to send us what the user inputs, all the other logic should be done by the backend. Also remember to update the readme with these changes so frontend knows what to send